### PR TITLE
[TEST] fix: update thumbnails

### LINKED ISSUE #736: Update recently fixed thumbnails
Update the thumbnails from this design issue: https://github.com/carbon-design-system/carbon-platform/issues/703

This should be done in the repos that have indexed these assets.

In the short term, also add them to the platform repo since we aren't rendering remote SVGs yet.

Also, as a best practice, all thumbnails should be saved in Box: https://ibm.ent.box.com/folder/158542444953

We may need to move thumbnails around in the future and we can avoid future conflicts if Box is our source of truth. CC @jeanservaas

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #736 

This PR updates a total of 21 thumbnails for the following repos: 

- Angular: https://github.com/IBM/carbon-components-angular/pull/2110/

- Svelte: https://github.com/carbon-design-system/carbon-components-svelte/pull/1307

- Carbon React: https://github.com/carbon-design-system/carbon/pull/11429

- Carbon for IBM.com: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/8867

### TESTING: 

- I'm not sure how to reproduce this 👀 for testing

- Make sure all images look good

### CODE REVIEW

**Comment by alisonjoseph** [comment]
Is the name change for unstable-text to text-unstable.svg intentional?

**Comment by francinelucca** [comment]
The way we're currently reading thumbnail images (locally), we're looking up the file by the asset slug: 
image
so I think we are constrained to save the images as the asset-id for now. (this will no longer be a problem when we implement #297)

**Review by francinelucca — COMMENTED** [review]
LGTM except for unstable-text thumbnail

**Comment by andreancardona** [comment]
@alisonjoseph @francinelucca should be fixed!!

**Review by francinelucca — DISMISSED** [review]
LGTM! 
image
Let's revert the changes in services/web-app/data/libraries.js and this is good to approve :)

**Review by francinelucca — COMMENTED** [review]
.

**Comment by alisonjoseph** [comment]
Looks good!

**Comment by andreancardona** [comment]
adding the do not merge label until we change code so that it's ready for prod

**Review by francinelucca — DISMISSED** [review]

File: services/web-app/data/libraries.js [hunk 1]
```diff
@@ -417,7 +417,7 @@ const devLibraries = {
   },
   'carbon-components-angular': {
     host: 'github.com',
-    org: 'IBM',
+    org:  'IBM',
```
Comment (alisonjoseph) [inline services/web-app/data/libraries.js:420]:
```suggestion
    org: 'IBM',
```

**Review by alisonjoseph — APPROVED** [review]

**Review by francinelucca — APPROVED** [review]

**Pull request merged**